docs/pages/upgrading/reference.mdx

```diff
@@ -0,0 +1,502 @@
+```code
+$ zypper removelock "teleport-ent"
+$ zypper install --repo="teleport" "teleport-ent-<target-version>"
+$ zypper addlock "teleport-ent"
+```
+
+## Automatic update limitations
+
+Automatic updates are not available in all Teleport editions and installation
+methods. If you cannot use automatic updates, read [Manual
+updates](#manual-updates) for possible steps.
+
+### Automatic updates with Teleport Community Edition
+
+Automatic updates is not currently supported with the community editions of Teleport.
+Ensure that you are using the Teleport Enterprise edition of the `teleport-kube-agent`
+chart. You should see the following when you query your `teleport-kube-agent` release:
+
+```code
+$ helm -n <Var name="teleport" /> get values <Var name="teleport-agent" /> -o json | jq '.enterprise'
+true
+```
+
+### Automatic updates with direct binary installation
+
+Automatic updates is not currently supported with the direct binary installation method.
+Automatic updates is only supported with installations via the `apt`, `yum`, and
+`zypper` package managers.
+
+## Automatic updates with GitOps tools
+
+Automatic updates are incompatible with some GitOps tools used for continuous
+deployment.  The `teleport-kube-agent` Helm chart owns the version of the
+`teleport-agent` resource, so when the `teleport-agent-updater` modifies the
+image version of the `teleport-agent` resource, the GitOps tool will detect a
+drift or a diff in the `teleport-agent` resource.
+
+### ArgoCD deployments
+
+After an automatic update, ArgoCD reports the `teleport-agent` resource as `OutOfSync`.
+As a workaround to this problem use a [Diff Customization](https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#diffing-customization)
+to ignore the difference in image version. Here is an example deployment using the
+name `teleport-agent` and namespace `teleport`.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: teleport-agent
+  namespace: teleport
+spec
+  ignoreDifferences:
+  - group: apps
+    kind: StatefulSet
+    name: teleport-agent
+    namespace: teleport
+    jqPathExpressions:
+    - .spec.template.spec.containers[] | select(.name == "teleport").image
+...
+```
+
+### FluxCD deployments
+
+After an automatic update, FluxCD reports a `DriftDetected` event. As a workaround
+to this problem modify the [drift detection](https://fluxcd.io/flux/components/helm/helmreleases/#drift-detection)
+configuration to ignore the difference in image version. Here is an example deployment
+using the name `teleport-agent` and namespace `teleport`.
+
+```yaml
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: teleport-agent
+  namespace: teleport
+spec
+  driftDetection:
+    mode: enabled
+    ignore:
+    - paths: ["/spec/template/spec/containers/0/image"]
+      target:
+        kind: StatefulSet
+        name: teleport-agent
+        namespace: teleport
+...
+```
+
+## Self-hosted automatic updates
```

Comment:
Why is this not a separate guide, peer to Cloud automatic upgrades? Seems weird to have this hidden inside reference.